src/EnvelopeCreator/CreateDocument.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the DocusignBundle.
+ *
+ * (c) Grégoire Hébert <gregoire@les-tilleuls.coop>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace DocusignBundle\EnvelopeCreator;
+
+use DocuSign\eSign\Model;
+use DocusignBundle\EnvelopeBuilderInterface;
+use League\Flysystem\FileNotFoundException;
+
+final class CreateDocument implements EnvelopeBuilderCallableInterface
+{
+    public function __invoke(EnvelopeBuilderInterface $envelopeBuilder, array $context = []): void
```

Comment:
Since I have an `EnvelopeBuilder` per signature configured, 
Then I cannot have a unique createDocument anymore.
I need t have 1 per signature configured. Not with this way.
WDYT ? Should I do it anyway? Or should I keep it as it is?

Reply:
Yep I think 1 CreateDocument per signature could be interesting